packages/wallets/src/components/WalletModal/index.tsx

```diff
@@ -0,0 +1,4 @@
+import WalletModal from './WalletModal';
+import './WalletModal.scss';
```

Comment:
@yashim-deriv 
Do we need to add scss files inside the components instead of index.ts file.
Suggestion needed.

Reply:
Oh yeah export { default as } 
@adrienne-deriv

Reply:
If there's only 1 component being exported in the folder, usually we can put the css import statement in the index.ts file right?